component-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -48,7 +49,7 @@
         <!-- Hyperdrive -->
         <dependency>
             <groupId>za.co.absa.hyperdrive</groupId>
-            <artifactId>api</artifactId>
+            <artifactId>api_\${scala.compat.version}</artifactId>
```

Comment:
what is the `\` for?

Reply:
This is a pom file for an archetype that generates the pom file. In this case, I want the output to be `<artifactId>api_${scala.compat.version}</artifactId>` and not `<artifactId>api_2.11</artifactId>`

Reply:
aah, an actual escape. Thank you!